frontend/src/components/table/Table.js

```diff
@@ -20,6 +20,7 @@ export default class Table extends PureComponent {
       listenOnKeys: true,
       tableRefreshToggle: false,
     };
+    this.multiSelStart = null;
```

Comment:
find a better name for `multiSelStart`.
Maybe something like `multiSelectionStartId`? (or something which when i read I know what's that).

Reply:
put `multiSelectionStartIdx`